.github/workflows/release-binaries-setup-stage/action.yml

```
@@ -22,7 +22,16 @@ runs:
   using: "composite"
   steps:
     - name: Install Ninja
+      # This doesn't seem to work on Ubuntu any more, so just use apt-get
+      if: runner.os != 'Linux'
       uses: llvm/actions/install-ninja@22e9f909d35b50bd1181709564bfe816eaeaae81 # main
+
+    - name: Install Ninja (Linux)
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install ninja-build
```

It seems like this should work still, no?
```suggestion
    - name: Update apt
      # Needed for the install-ninja action to work on Ubuntu for some reason.
      if: runner.os == 'Linux'
      run: sudo apt-get update

    - name: Install Ninja
      uses: llvm/actions/install-ninja@22e9f909d35b50bd1181709564bfe816eaeaae81 # main
```

---

This is working now, so I dropped this part of the change.